.github/workflows/docker-image.yml

```diff
@@ -18,9 +18,19 @@ jobs:
     - name: Get current tag
       id: vars
       run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+    - name: Extract Major and Minor Versions
+      id: version
+      run: |
+        FULL_VERSION=${{ steps.vars.outputs.tag }}
+        MAJOR_VERSION=$(echo $FULL_VERSION | cut -d '.' -f 1)
```

Comment:
Thank you for elaborating on the other comments!

Before I approve, I have a final thing to decide on. I wonder if we should remove major version tagging.

I don't think it makes much sense based on our current releases.

Maybe @generall has a good opinion on this?

Reply:
Yes makes sense, but I feel maintaining major tags is a standard practice & usually beneficial long-term. Your call, though.